supplementary_style_guide/glossary_terms_conventions/general_conventions/d.adoc

```diff
@@ -331,7 +331,7 @@ The API object for a deployment can be either a Kubernetes-native `Deployment` o
 [discrete]
 [[devops-n]]
 ==== image:images/yes.png[yes] DevOps (noun)
-*Description*: "DevOps" is a combination of "Development" and "Operations". It refers to a specific method or organizational approach where developers and IT operations staff work together to create the applications that run the business.
+*Description*: _DevOps_ is a combination of Development and Operations. It refers to a specific method or organizational approach where developers and IT operations staff work together to create the applications that run the business.
```

Comment:
I think the quotes around "Development" and "Operations" were probably appropriate

Reply:
:heavy_check_mark: